modules/manifold/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@mlvis/manifold.ml",
```

Comment:
Could we check if `@mlvis/manifold` is available? If so we don't need .ml here

Reply:
it is still available.

Reply:
Then maybe change to `@mlvis/manifold`